README.md

```diff
@@ -111,6 +111,69 @@ describe('application launch', function () {
 })
 ```
 
+### With AVA
+
+Spectron also works with [AVA](https://github.com/sindresorhus/ava) that allows you to write your test code in ES2015 without extra support.
+
+```js
+'use strict';
+
+import test from 'ava';
+import {Application} from 'spectron';
+import path from 'path';
+
+test.beforeEach(t => {
+	t.context.app = new Application({
```

Comment:
Looks like this is using hard tabs, mind using a two space indent to match the other code blocks in this file?